Dockerfile

```diff
@@ -1,27 +1,48 @@
-FROM node:20
+FROM node:24-alpine
```

Comment:
I am thinking about, whether we should use the exact same base image for the dev image - then a developer would register any problems in his dev setup before trying with the prod image. What do you think?